src/util/util.js

```diff
@@ -32,7 +32,7 @@ function resolveURLTemplates(URLTemplates, macros = {}, options = {}) {
   }
 
   // Calc random/time based macros
-  macros['CACHEBUSTING'] = leftpad(
+  macros['CACHEBUSTING'] = addLeadingZeros(
```

Comment:
Won't `Math.round(Math.random() * 1.0e8).toString().padStart(8, '0')` do the job here ? 🤔

Reply:
I didn't know such a thing existed :o